tox.ini

```diff
@@ -26,13 +28,13 @@ commands =
 [testenv:towncrier]
 skip_install = True
 deps =
-    towncrier == 21.9.0
+    towncrier ~= 22.8
```

Comment:
This upgrade seems the most capricious...however, I ran `tox -e towncrier` and it deleted the news fragments and updated release docs.